src/main/resources/templates/smallfull/accountsReferenceDateQuestion.html

```diff
@@ -0,0 +1,40 @@
+
+<!DOCTYPE html>
+
+<html
+        xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.w3.org/1999/xhtml"
+        layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title id="page-title-heading">
+        Accounts Reference Date Question
+    </title>
+</head>
+<div id="accounts-reference-date-question-main-content" layout:fragment="content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form id="accounts-reference-date-question--form" class="form" th:action="@{''}" th:object="${accountsReferenceDateQuestion}" method="post">
+
+                <div th:replace="fragments/globalErrors :: globalErrors"></div>
+
+                <input th:field="${accountsReferenceDateQuestion.periodStartOn}" type="hidden">
```

Comment:
Nitpick suggestion, maybe preface this with an inline HTML comment so it's obvious what this is for to other devs?